client/src/pages/Ingestion/components/Metadata/Control/GenerateModelControl.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
I'll be honest here -- I don't love the name of this control.  It sounds like something used to control generation of models.  While it's true that scene generation produces models, this UI is really about controlling scene generation. Please consider a rename.

Reply:
How does SceneGenerateWorkflowControl sound?